allensdk/api/cloud_cache/cloud_cache.py

```diff
@@ -184,11 +182,14 @@ def load_manifest(self, manifest_name: str):
                              "for this dataset:\n"
                              f"{self.manifest_file_names}")
 
-        with io.BytesIO() as stream:
-            self._download_manifest(manifest_name, stream)
+        filepath = os.path.join(self._cache_dir, manifest_name)
+        if not os.path.exists(filepath):
```

Comment:
Because I'm *really* worried about users accidentally touching the manifest:

Can we have this method start by trying to download the manifest file from S3 and saving it to `filepath`. If it finds that there is no internet connection, it falls back on loading a file from a pre-existing `download`. If that does not exist, it informs the user?

That way, if the user has an internet connection, they are guaranteed to get a correct manifest. We only have to trust locally downloaded copies if there is no internet connection.